scapy/arch/pcapdnet.py

```diff
@@ -548,19 +631,23 @@ def get_if_raw_addr(ifname):
             i = dnet.intf()
             try:
                 return i.get(ifname)["addr"].data
-            except OSError:
+            except (OSError, KeyError):
```

Comment:
I may include this fix in a different PR if really needed

Reply:
Can you move it to another PR ? Thanks.